gazelle/python/generate.go

```diff
@@ -485,6 +488,44 @@ func (py *Python) GenerateRules(args language.GenerateArgs) language.GenerateRes
 	return result
 }
 
+// getRulesWithInvalidSrcs checks existing Python rules in the BUILD file and return the rules with invalid srcs.
+func (py *Python) getRulesWithInvalidSrcs(args language.GenerateArgs) (invalidRules []*rule.Rule) {
+	if args.File == nil {
+		return
+	}
+	filesMap := make(map[string]struct{})
+	for _, file := range args.RegularFiles {
+		filesMap[file] = struct{}{}
+	}
+	for _, file := range args.GenFiles {
+		filesMap[file] = struct{}{}
+	}
+
+	isTarget := func(src string) bool {
+		return strings.HasPrefix(src, "@") || strings.HasPrefix(src, "//") || strings.HasPrefix(src, ":")
+	}
+	for _, existingRule := range args.File.Rules {
+		if existingRule.Kind() != pyBinaryKind {
+			continue
+		}
+		allInvalidSrcs := true
```

Comment:
I would avoid double negative in the code. So instead of "allInvalidSrcs == false", it's easier to understand if we say "hasValidSrcs == true"